runtime/reference/node.md

```diff
@@ -182,13 +239,13 @@ of moving components in your project. Deno also has a more secure runtime, with
 [runtime permissions](../basics/permissions.md) that allow you to control what
 your code can access.
 
-#### deno.json (optional)
+#### deno.json
 
 Deno has its own config file, `deno.json` or `deno.jsonc`, which can be used to
 configure your project. You can use it to define tasks, dependencies, path
 mappings, and other runtime configurations.
 
-#### Migrating npm scripts to deno.json (optional)
+#### Migrating npm scripts to deno.json
```

Comment:
"Migrating" seems like a bad word to use.

Does this section even belong here in the `node.md`?